packages/dnb-eufemia/src/components/pagination/PaginationDocs.ts

```diff
@@ -6,6 +6,11 @@ export const PaginationProperties: PropertiesTableProps = {
     type: ['pagination', 'infinity'],
     status: 'optional',
   },
+  paginationLayout: {
```

Comment:
```suggestion
  paginationBarLayout: {
```

Should we name it `paginationLayout` or `paginationBarLayout`?
I see that it only changes the layout of the pagination bar, so perhaps we should rename `paginationLayout` to `paginationBarLayout`?

Worth a consideration 📿

Reply:
Yes, that is better – I think too.